password_security/controllers/main.py

```diff
@@ -44,6 +45,7 @@ def web_login(self, *args, **kw):
             request.params['password']
         )
         if not uid:
+            request.uid = old_uid
```

Comment:
@nguyenductamlhp - It looks to me like the code currently works, without the sudo. How can I replicate the security issues that you're referring to?